### PR TITLE
test: add unit test for ExplorerCard

### PR DESCRIPTION
**Pull request by gitstart**
Part of #2200 

Screenshot 2023-08-28 at 2 25 00 PM

---
This code was written and reviewed by GitStart Community. Growing great engineers, one PR at a time.

### CODE REVIEW

File: frontend/src/components/ExplorerCard/explorerCard.test.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+
+import ExplorerCard from './index';
+
+const hrefLink =
```
Comment (palashgdev) [inline frontend/src/components/ExplorerCard/explorerCard.test.tsx:5]: can we pull out in form of constant and use it here

File: frontend/src/components/ExplorerCard/explorerCard.test.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+type RenderExplorerCard = () => ReturnType<typeof render>;
+
+const renderExplorerCard: RenderExplorerCard = () =>
+	render(
+		<ExplorerCard>
+			<div>Test Children</div>
+		</ExplorerCard>,
+	);
+
+describe('Test ExplorerCard', () => {
```
Comment (palashgdev) [inline frontend/src/components/ExplorerCard/explorerCard.test.tsx:19]: components/ExplorerCard

File: frontend/src/constants/app.ts [hunk 3]
```diff
@@ -13,3 +13,6 @@ export const SIGNOZ_UPGRADE_PLAN_URL =
 	'https://upgrade.signoz.io/upgrade-from-app';
 
 export const DASHBOARD_TIME_IN_DURATION = 'refreshInterval';
+
+export const QUERY_BUILDER_URL =
```
Comment (gitstart) [inline frontend/src/constants/app.ts:17]: Hi @palashgdev, we moved the URL to constants but are not sure if this is the right file to move to

Comment (palashgdev) [reply frontend/src/constants/app.ts:17]: @gitstart just move under Explorer Card constant not over global one

**Comment by gitstart** [comment]
Closing this since the component has been updated and these tests are no longer relevant.

**Pull request closed without merging**